subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -185,26 +196,13 @@ struct node_rx_hdr {
 
 	enum node_rx_type   type;
```

Comment:
A comment on `type` would be nice to explain that `type` and `rx_ftr` forms a tagged union. (`rx_ftr` is already currently an implicit union).
And then move `type` down to be near footer.

Reply:
This is form previous code. We can fix this in a later PR